python/raydp/torch/estimator.py

```diff
@@ -233,12 +235,20 @@ def train_func(config):
         for epoch in range(config["num_epochs"]):
             train_res, train_loss = TorchEstimator.train_epoch(train_dataset, model, loss,
                                                                 optimizer, metrics, lr_scheduler)
-            train.report(epoch=epoch, train_res=train_res, train_loss=train_loss)
+            session.report(dict(epoch=epoch, train_res=train_res, train_loss=train_loss))
             if config["evaluate"]:
-                evaluate_res, evaluate_loss = TorchEstimator.evaluate_epoch(evaluate_dataset,
+                eval_res, evaluate_loss = TorchEstimator.evaluate_epoch(evaluate_dataset,
                                                                             model, loss, metrics)
-                train.report(epoch=epoch, evaluate_res=evaluate_res, test_loss=evaluate_loss)
+                session.report(dict(epoch=epoch, eval_res=eval_res, test_loss=evaluate_loss))
                 loss_results.append(evaluate_loss)
+        if hasattr(model, "module"):
+            states = model.module.state_dict()
+        else:
+            # if num_workers = 1, model is not wrapped
+            states = model.state_dict()
+        session.report({}, checkpoint=Checkpoint.from_dict({
```

Comment:
This saves the checkpoint for every epoch. Is this a default behavior? Otherwise we also don't need to do it by default.

Reply:
I tested removing this, but the checkpoint included in the returned result does not contain anything.

Reply:
Where is checkpoint stored? Are anything in the path?

Reply:
It is stored in ~/ray_results, there are some pickled binary files. But when I use TorchCheckpoint.from_directory to load it, get_model does not work.